libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -125,7 +125,8 @@ void AP_Mount_Backend::update_targets_from_rc()
 // returns the angle (degrees*100) that the RC_Channel input is receiving
 int32_t AP_Mount_Backend::angle_input(const RC_Channel* rc, int16_t angle_min, int16_t angle_max)
 {
-    return (rc->get_reverse() ? -1 : 1) * (rc->get_radio_in() - rc->get_radio_min()) 
+    int16_t radio_in = constrain_int16(rc->get_radio_in(),  rc->get_radio_min(),  rc->get_radio_max());
+    return (rc->get_reverse() ? -1 : 1) * (radio_in - rc->get_radio_min())
```

Comment:
Is there a reason why we can't just use the `norm_input` or `percent_input` methods here? The control_in could also be used if we set the RC channel type and range.

Reply:
There's probably no reason except that we would need to be very careful to get it right and it wouldn't actually make things better.  This is the minimum fix which will make it easy to merge into Copter-3.6.

Reply:
I've added a commit which changes it to use norm_input.  It's not quite a non-functional change because the new calculation uses the RCx_TRIM while the old one did not.